crates/modules/smtp-notifier/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "smtp-notifier"
+version.workspace = true
+license.workspace = true
+edition.workspace = true
+repository.workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+pulsar-core = { path = "../../pulsar-core" }
+
+tokio = { version = "1", features = ["full"] }
```

Comment:
Hmm... can we double-check whether we really need the `full` set of features? Maybe it's possible to use just specific ones?